config/settings/container.py

```diff
@@ -5,3 +5,9 @@
 DATABASES["default"]["NAME"] = Path("/") / "db" / "db.sqlite3"
 STATIC_ROOT = "/static/"
 MEDIA_ROOT = "/images/"
+
```

Comment:
Is this not needed in all cases? Could it go into `base.py`? ...okay, okay, I know! Work in progress, here. Have fun with it!

Reply:
Okay, I am just struggling with `MEDIA_ROOT` on other fronts: When using the dev server, the uploads are put in `<workdir>/uploads/`, when I start the docker image, they are uploaded to `<workdir>/cpmonitor/images/uploads` (I assume, but have not tested. Therefore, I would expect that in the dev server case it would be good to add something like

```python
MEDIA_ROOT = BASE_DIR / "cpmonitor" / "images"
```

I will try that anyway now... Jep. Works. Some more [trickery](https://github.com/GermanZero-de/klimaschutzmonitor/pull/143/commits/030f28f275acb2d05c86cac53a47e2ead6d44193) was needed, but okay.